packages/hmr-plugin/src/symbols.ts

```diff
@@ -23,6 +23,14 @@ export interface NgxsHmrLifeCycle<T = NgxsHmrSnapshot> {
 export type HmrCallback<T> = (ctx: StateContext<T>, state: Partial<T>) => void;
 export type BootstrapModuleFn<T = any> = () => Promise<NgModuleRef<T>>;
 
+export interface WebpackHotApi<T = any> {
```

Comment:
There is no need to separate this interface from the `WebpackHotModule` interface or export it.
The outside world will only ever need to interact with the `WebpackHotModule` interface.